frontend/src/lib/doc-util.tsx

```diff
@@ -13,24 +13,24 @@ export const OCP_DOC_BASE_PATH = 'https://docs.openshift.com/container-platform'
-    CREATE_CONNECTION_VMWARE: `${DOC_BASE_PATH}/credentials/credentials#creating-a-credential-for-vmware-vsphere`,
-    CREATE_CONNECTION_OPENSTACK: `${DOC_BASE_PATH}/credentials/credentials#creating-a-credential-for-openstack`,
-    CREATE_CONNECTION_VIRTUALIZATION: `${DOC_BASE_PATH}/credentials/credentials#creating-a-credential-for-virtualization`,
-    CREATE_CONNECTION_BAREMETAL: `${DOC_BASE_PATH}/credentials/credentials#creating-a-credential-for-bare-metal`,
-    CREATE_CONNECTION_ANSIBLE: `${DOC_BASE_PATH}/credentials/credentials#creating-a-credential-for-ansible`,
-    CREATE_CONNECTION_REDHATCLOUD: `${DOC_BASE_PATH}/credentials/credentials#creating-a-credential-for-openshift-cluster-manager`,
-    CREATE_CONNECTION_PROXY: `${DOC_BASE_PATH}/credentials/credentials#proxy`,
-    CREATE_CONNECTION_PROXY_VMWARE: `${DOC_BASE_PATH}/credentials/credentials#proxy-vm`,
-    CREATE_CONNECTION_PROXY_AWS: `${DOC_BASE_PATH}/credentials/credentials#proxy-aws`,
-    CREATE_CONNECTION_PROXY_AZURE: `${DOC_BASE_PATH}/credentials/credentials#proxy-azure`,
-    CREATE_CONNECTION_PROXY_BAREMETAL: `${DOC_BASE_PATH}/credentials/credentials#proxy-bare`,
-    CREATE_CONNECTION_PROXY_GCP: `${DOC_BASE_PATH}/credentials/credentials#proxy-google`,
-    CREATE_CONNECTION_PROXY_OPENSTACK: `${DOC_BASE_PATH}/credentials/credentials#proxy-openstack`,
-    CREATE_CONNECTION_PROXY_VIRTUALIZATION: `${DOC_BASE_PATH}/credentials/credentials#proxy-virtualization`,
+    CREATE_CONNECTION: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials`,
+    CREATE_CONNECTION_AWS: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-amazon-web-services`,
+    CREATE_CONNECTION_AZURE: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-microsoft-azure`,
+    CREATE_CONNECTION_GCP: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-google-cloud-platform`,
+    CREATE_CONNECTION_VMWARE: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-vmware-vsphere`,
+    CREATE_CONNECTION_OPENSTACK: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-openstack`,
+    CREATE_CONNECTION_VIRTUALIZATION: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-virtualization`,
+    CREATE_CONNECTION_BAREMETAL: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-bare-metal`,
+    CREATE_CONNECTION_ANSIBLE: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-ansible`,
+    CREATE_CONNECTION_REDHATCLOUD: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-openshift-cluster-manager`,
+    CREATE_CONNECTION_PROXY: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#proxy`,
```

Comment:
I am not sure about proxy here. @cadawson will take this over today (Friday)

Reply:
I think this one was split by platform.